test/users/signs_up.test.js

```diff
@@ -0,0 +1,77 @@
+const faker = require('faker');
+const supertest = require('supertest');
+
+const app = require('../../app');
+const db = require('../../app/models/index');
+const logger = require('../../app/logger');
+
+describe('Users - Sign Up', () => {
+  beforeEach(() => {
+    const data = {
+      name: faker.name.findName(),
+      last_name: faker.name.lastName(),
+      email: 'first.user@wolox.co',
+      password: faker.internet.password()
+    };
+    db.User.create(data)
+      .then()
+      .catch(err => {
+        logger.error(err.errors);
+      });
+  });
+
+  it('Response status 200 - User created correctly', async done => {
+    const data = {
```

Comment:
I think that the code to generate fake data of a user is being repeated several times in the code, you do not think it would be better to unify it in another part something like

tests / factory / users.js